apps/web/lib/api/links/process-link.ts

```diff
@@ -374,7 +374,6 @@ export async function processLink<T extends Record<string, any>>({
       key,
       // we're redefining these fields because they're processed in the function
       url,
-      createdAt: createdAt ? new Date(createdAt) : undefined,
```

Comment:
@steven-tey I think we still need to pass `createdAt` here so that the processed link retains it from the original payload

Reply:
so I think we're not supposed to be able to pass `createdAt` since it should be the date the link gets created in Dub. the only exception is in link imports, so we'll need to handle that separately

Reply:
Okay I guess we can set it after calling `processLink` or something